pkg/volume/cinder/cinder.go

```diff
@@ -47,7 +48,8 @@ type CinderProvider interface {
 	AttachDisk(instanceID string, diskName string) (string, error)
 	DetachDisk(instanceID string, partialDiskId string) error
 	DeleteVolume(volumeName string) error
-	CreateVolume(name string, size int, vtype, availability string, tags *map[string]string) (volumeName string, err error)
+	CreateVolume(name string, size int, vtype, availability string, tags *map[string]string) (volumeName string, volumeZone string, err error)
```

Comment:
Again, give `availability` and `volumeZone` names that make it clearer that they're the same type of value.